src/views/assets/Asset/AssetCreateUpdate/BaseAssetCreateUpdate.vue

```diff
@@ -141,22 +134,9 @@ export default {
       }
       this.iConfig = config
     },
-    async getMeta() {
-      let meta = await this.$store.dispatch('common/getDrawerActionMeta')
-      if (!meta || !meta.action) {
-        meta = {
-          action: 'create',
-          platform: this.$route.query.platform || '',
-          type: this.$route.query.type || '',
-          category: this.$route.query.category || '',
-          node: this.$route.query.node || this.$route.query.node_id || ''
-        }
-      }
-      this.meta = meta
-    },
     async setInitial() {
       const { defaultConfig } = this
-      const { node, platform } = this.meta
+      const { node, platform } = this.$route.query
       const nodesInitial = node ? [node] : []
       const platformId = this.changePlatformID || this.$route.query.platform || (platform || 'Linux')
       const url = `/api/v1/assets/platforms/${platformId}/`
```

Comment:
There are no major code changes between the two versions you provided. The only difference is that we moved the defaultConfig fields to a separate object with fieldsMeta which could improve readability. For the rest of the code there might be some small optimizations that can be made like reducing number of steps while handling the same request.

In general, though, these codes seem quite well-documented and structured already without too much room for optimization based on current practices.